OpenRA.Mods.Common/Warheads/HealthPercentageDamageWarhead.cs

```diff
@@ -22,6 +23,10 @@ public class HealthPercentageDamageWarhead : DamageWarhead
 		public override void DoImpact(WPos pos, Actor firedBy, IEnumerable<int> damageModifiers)
 		{
 			var world = firedBy.World;
+
+			if (world.AllowDevCommands && world.LocalPlayer.PlayerActor.Trait<DeveloperMode>().ShowCombatGeometry)
```

Comment:
This crashes in replays because `LocalPlayer` is null.
Could just go for a null check because these aren't supposed to be visible in replays and for spectators in live games.